### ISSUE TITLE
Add support for Slovenian eID

### ISSUE THREAD

**Issue by craftbyte** [web-eid/libelectronic-id#90]
Would it be possible to support (either on APDU or PKCS#11 level) Slovenian eIDs? You can find documentation [here](https://github.com/OpenSC/OpenSC/wiki/Slovenian-EID) and [here](https://github.com/OpenSC/OpenSC/issues/2564#issuecomment-1196015941). The official library is available [here](https://www.si-trust.gov.si/sl/eoi/programska-oprema-idprotect-client/). If you need any extra documentation/support, I can also provide a contact at the government department that oversees eIDs.

**Comment by mrts** [comment]
Yes, we can add Slovenian eID support. We need your help with testing, we'll get back to this as time allows.

**Comment by mrts** [comment]
Can you perhaps provide at least the initial version yourself? It should be relatively easy to start with PKCS#11 level, see e.g. [Check eID support PR](https://github.com/web-eid/libelectronic-id/pull/52/files).

**Comment by mrts** [comment]
Here are step-by-step instructions for adding Slovenian eID support at the PKCS#11 level:

---

#### 1. Modify `electronic-id.hpp`

- Locate the `ElectronicID` class and add an entry for Slovenian eID.
  ```cpp
  SloEID, // Add this line below the last eID type
  ```

---

#### 2. Update `electronic-id.cpp`

- Extend the `SUPPORTED_ATRS` map with the Slovenian eID ATR and its corresponding constructor. Replace `<ATR_VALUES>` with the Slovenian eID ATR values.
  ```cpp
  // SloEID
  {{<ATR_VALUES>},
   constructor<Pkcs11ElectronicIDType::SloEID>},
  ```

---

#### 3. Extend `Pkcs11ElectronicID.cpp`

- Add a helper function to define the PKCS#11 module path for Slovenian eID.
  ```cpp
  inline fs::path slovenianPkcs11ModulePath()
  {
  #ifdef _WIN32
      return system32Path() / L"sloveniankp11.dll";
  #elif defined __APPLE__
      return "/path/to/SLOEID/libsloveniankp11.dylib";
  #else // Linux
      return "/path/to/libsloveniankp11.so";
  #endif
  }
  ```

- Add the Slovenian eID configuration to the `SUPPORTED_PKCS11_MODULES` map.
  ```cpp
  {Pkcs11ElectronicIDType::SloEID,
   {
       "Slovenian eID (PKCS#11)"s,  // name
       ElectronicID::Type::SloEID,  // type
       slovenianPkcs11ModulePath().make_preferred(), // path
       
       JsonWebSignatureAlgorithm::RS256, // authSignatureAlgorithm
       RSA_SIGNATURE_ALGOS(),            // supportedSigningAlgorithms
       3,
       true,
       false,
   }},
  ```

---

#### 4. Extend `Pkcs11ElectronicID.hpp`

- Add `SloEID` to the `Pkcs11ElectronicIDType` enum.
  ```cpp
  SloEID, // Add this below the last entry
  ```
